### DISCUSSION TITLE
triage does not work?

### DISCUSSION THREAD

**Discussion by Tagamoga** [community/community#22931] [New to GitHub]
<p>Hi…</p>
<p>I newly created an Issue. Clicked on “(awaing triage)” and landed in project view. Now I tried to drag an drop this Item by mouse to one of my columns.</p>
<p>The item stays at the beginning in the columns, but after refreshing the Issues is still wainting for triage and is not in any columns.</p>
<p>How do I sort Issues into the colums?</p>
<p>Thanks for help!</p>

**Comment by ihalilaltun** [comment]
<p>I am also getting the same problem.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/165840229556cbf2fbd5433409e7a33426939914.png" data-download-href="/uploads/short-url/3bFzYpSGkfnOdYnY5UuIFYwNjDu.png?dl=1" title="Screen Shot 2018-06-18 at 15.24.25.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/5309694/181073740-8e7fc1dd-d1f2-4161-accc-26ef69dc1127.png" alt="Screen Shot 2018-06-18 at 15.24.25.png" data-base62-sha1="3bFzYpSGkfnOdYnY5UuIFYwNjDu" width="690" height="152" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/165840229556cbf2fbd5433409e7a33426939914_2_690x152.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/165840229556cbf2fbd5433409e7a33426939914_2_1035x228.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/165840229556cbf2fbd5433409e7a33426939914_2_1380x304.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/165840229556cbf2fbd5433409e7a33426939914_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-06-18 at 15.24.25.png</span><span class="informations">1659×366 60.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ihalilaltun** [comment, marked as answer]
<p>I’ve contacted to support group on GitHub, and they’ve already fixed it :slight_smile:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/544364686518928dceac6006debc382183d690f7.png" data-download-href="/uploads/short-url/c1quFxNqi2WlDGIOwBzw7eIYT55.png?dl=1" title="Screen Shot 2018-06-18 at 17.52.34.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/5309694/181073744-a4acc95a-9738-4cf1-99e1-b9d46e1021d9.png" alt="Screen Shot 2018-06-18 at 17.52.34.png" data-base62-sha1="c1quFxNqi2WlDGIOwBzw7eIYT55" width="690" height="148" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/544364686518928dceac6006debc382183d690f7_2_690x148.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/544364686518928dceac6006debc382183d690f7_2_1035x222.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/5/544364686518928dceac6006debc382183d690f7.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/544364686518928dceac6006debc382183d690f7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-06-18 at 17.52.34.png</span><span class="informations">1108×238 21.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Tagamoga** [comment]
<p>Thank you very much!!!</p>
<p>It works!</p>
<p>:catvery-happy:</p>
